ios/MLCSwift/Sources/ObjC/include/LLMChat.h

```diff
@@ -80,4 +89,37 @@
  *  for debugging purposes.
  */
 - (void)evaluate;
+
+/**
+ * Unload the current image model and free all memory.
+ * @note This function is useful to get memory estimation before launch next model.
+ */
+- (void)unloadImageMod;
+
+/**
+ * Reload the image module to a new model.
+ *
+ * @param modelLib The name of the modelLib
+ * @param modelPath The path to the model artifacts.
+ */
+- (void)reloadImageMod:(NSString*)modelLib modelPath:(NSString*)modelPath;
+
+/**
+ * Reset the current image model.
+ */
+- (void)resetImageMod;
+
+/**
+ * @returns Runtime stats of the image encoding stage.
+ */
+- (NSString*)runtimeStatsTextImageMod;
+
+/**
+ * Prefill the LLM with the embedding of the input image.
+ *
+ * @param image The uploaded image.
+ * @param prevPlaceholder The previous placeholder in the prompt, i.e. <Img>.
```

Comment:
these parameters should be part of the LLMChat i think instead being customizable

Reply:
I think these are the placeholders to separate image embedding from the rest of text embedding. In MiniGPT, the placeholders are `<Img>` and `</Img> `, but I think it might vary from model to model? So I leave them as parameters here.